sql/api/src/main/scala/org/apache/spark/sql/catalyst/analysis/noSuchItemsExceptions.scala

```diff
@@ -52,6 +52,13 @@ class NoSuchDatabaseException private(
       messageParameters = Map("schemaName" -> quoteIdentifier(db)),
       cause = None)
   }
+
+  def this(db: Seq[String]) = {
```

Comment:
```suggestion
  def this(namespace: Seq[String]) = {
```

Reply:
shall we use `NoSuchNamespaceException`?

Reply:
Okay